.github/workflows/smoke-tests.yml

```
@@ -184,6 +185,7 @@ jobs:
           tags: smoke-test
           build-args: |
             go=${{ matrix.go }}
+            go_min=1.22
```

Since this ends up in the `go` directive, I believe it should be

```suggestion
            go_min=1.22.0
```

---

121d0e3debdae2c20fd20dcddbc2da9ce0a7481f